src/lib/yang/parser.lua

```diff
@@ -84,6 +92,13 @@ function Parser:check(expected)
    return false
 end
 
+function Parser:check_pattern(patten)
```

Comment:
Ugh yeah, sorry. My ability to spell has always been somewhat lacking, I shall try to remember this one.